src/components/SingleNotification.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Image } from "react-bootstrap";
+import { HeartFill, ArrowRepeat } from "react-bootstrap-icons";
+import styled from "styled-components";
+
+const StyledSection = styled.section`
+  transition: 0.3s;
+  :hover {
+    background-color: rgba(0, 0, 0, 0.03) !important;
+  }
+`;
+
+const StyledImage = styled(Image)`
+  width: 2.5rem;
+  height: 2.5rem;
+`;
+
+const liked = "вподобав(ла)";
+const retweeted = "ретвітнув(ла)";
+
```

Comment:
maybe i18n should be used for all new components

Reply:
Done.